### DISCUSSION TITLE
Archiving a website

### DISCUSSION THREAD

**Discussion by Funeoz** [community/community#23818] [Other Feature Feedback, Questions, & Ideas]
<p>I would like to know if I archive my repository that is currently a website with gh-pages, will the website still accessible after the archivation ?</p>

**Comment by samuel-ping** [comment, marked as answer]
<p>This is a bit late, but I just archived a repo that was published to gh-pages and it looks like it is still deployed with no issues.</p>

**Comment by Funeoz** [comment]
<p>Ok thanks. I’ll do it right now.</p>
